test/models/test_inspection.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+"""
+<Program Name>
+  test_inspection.py
+
+<Author>
+  Santiago Torres-Arias <santiago@nyu.edu>
+
+<Started>
+  Nov 18, 2016
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Test inspection class functions.
+
+"""
+
+import unittest
+from toto.models.layout import Layout, Inspection
+from toto.ssl_commons.exceptions import FormatError
+import toto.ssl_crypto
+
+class TestInspectionValidator(unittest.TestCase):
+  """Test verifylib.verify_delete_rule(rule, artifact_queue) """
+
+  def setUp(self):
+    """ Populate a base layout that we can use """
+    self.inspection = Inspection("some-inspection")
+
+  def test_wrong_type(self):
+    """ test the type field within Validate() """
+
+    with self.assertRaises(FormatError):
+      self.inspection._type = "wrong"
+      self.inspection._validate_type()
+      self.assertFalse(self.inspection.validate())
```

Comment:
I don't understand this. You never reach the `assertFalse` because the `FormatError` gets raised before. Please see my comment [below](https://github.com/in-toto/in-toto/pull/48/files#diff-cc71a9e93777c3c30e19a468fd9fede4R54) regarding Exception vs Bool strategy. The comment applies for most of the unittests below.